src/mscorlib/shared/System/Byte.cs

```diff
@@ -111,8 +111,20 @@ public static byte Parse(String s, NumberStyles style, IFormatProvider provider)
             return Parse(s, style, NumberFormatInfo.GetInstance(provider));
         }
 
+        public static byte Parse(ReadOnlySpan<char> s, NumberStyles style, IFormatProvider provider)
+        {
+            NumberFormatInfo.ValidateParseStyleInteger(style);
+            return Parse(s, style, NumberFormatInfo.GetInstance(provider));
+        }
+
         private static byte Parse(String s, NumberStyles style, NumberFormatInfo info)
         {
+            if (s == null) ThrowHelper.ThrowArgumentNullException(ExceptionArgument.s);
```

Comment:
No reason, other than when editing 12 types with a bazillion overloads, one can lose track of the pattern (especially when they're already inconsistent) :)  I'll review and tweak where necessary.